electrumsv/nodeapi.py

```diff
@@ -762,9 +763,14 @@ async def jsonrpc_gettransaction_async(request: web.Request, request_id: Request
     # Compatibility: Raises RPC_INVALID_PARAMETER if we were given unlisted named parameters.
     parameter_values = transform_parameters(request_id, [ "txid", "include_watchonly" ],
         parameters)
-    txid = parameter_values[0]
-    node_RPCTypeCheckArgument(request_id, txid, str)
-    node_ParseHexV(request_id, "txid", txid)
+    if len(parameter_values) < 1 or len(parameter_values) > 2:
+        raise web.HTTPInternalServerError(headers={"Content-Type": "application/json"},
+            text=json.dumps(ResponseDict(id=request_id, result=None,
+            error=ErrorDict(code=RPCError.INVALID_PARAMS,
+            message="Invalid parameters, see documentation for this call"))))
+
+    txid = get_string_parameter(request_id, parameter_values[0])
```

Comment:
Uh.. you've just broken this code. That code you replaced did something specific returning the errors of the existing node api and you replaced it with something that doesn't do that.

Reply:
I did a lot of manual testing against the node and the node never returns the response from `node_ParseHexV` or `node_RPCTypeCheckArgument` for `gettransaction` so I dropped them to make ESV and the Node exactly equivalent when providing a variation of different inputs for `txid`.